src/libsystemd/sd-event/sd-event.c

```diff
@@ -3116,7 +3116,9 @@ static int process_timer(
         return 0;
 }
 
-static int process_child(sd_event *e) {
+static int process_child(sd_event *e, int64_t priority_threshold, int64_t *ret_highest_priority) {
```

Comment:
`ref_highest_priority` actually returns the lowest priority ;}. And since this is static, maybe make the parameter input/output, int64_t *priority_threshold.

Reply:
> And since this is static, maybe make the parameter input/output, int64_t *priority_threshold.

`process_child()` must be called at least once with `threshold == INT64_MAX`. So, `process_epoll()` should not update `threshold`, and needs to take `threshold` and `*ret_min_threshold`. `process_child()` can directly update `threshold`, but for consistency with `process_epoll()`, I'd like to keep the current form; it takes `threshold` and `*ret_min_threshold`.